src/styles/blocks/centre.scss

```diff
@@ -0,0 +1,11 @@
+.stop-watch__centre {
```

Comment:
the same - this is element of block "stop-watch", don't move it to separate file